actions.yaml

```diff
@@ -3,3 +3,14 @@
 
 restart:
   description: Restart the Temporal server.
+
+create-authorization-model:
```

Comment:
If it helps, I struggled to figure out how to get a big yaml file into the parameter. The following did the trick
`juju run-action <app>/leader create-authorization-model --params auth_model.yaml --wait`
and then `auth_model.yaml ` looked like the following
```
model: >
  {
    "type_definitions": [],
    ...
  }
```

Reply:
I've also found this, which I will include in the deployment instructions in a future PR:
```
juju run <app>/leader create-authorization-model  model="$(<temporal_auth_model.json)" --string-args=true
```